src/components/Calculator.js

```diff
@@ -1,61 +1,47 @@
-import React, { PureComponent } from 'react';
+import React, { useState, useEffect } from 'react';
 import Display from './Display';
 import Button from './Button';
 import '../styles/Calculator.scss';
 import calculate from '../logic/calculate';
+import buttons from './buttons';
 
-export default class Calculator extends PureComponent {
-  constructor(props) {
-    super(props);
+function Calculator() {
```

Comment:
- [ ] please make sure to use ES6 this includes the use of arrow functions instead of the `function` keyword

Reply:
@mikethreels, really? You required changes for a `function Calculator() { ... }` instead of `const Calculator = () => { ... }`? :raised_eyebrow:  Was that the essence of this task? Because I pretty clearly read the requirements stating:

### Project requirements

- Refactor the Calculator component (`components/Calculator.js`) from a class-based component to a functional component with _hooks_.
- After the refactor is done, test the app in the browser. It should keep the same functionality.

I'm pretty confused, then, which of those *requirements* did I not fulfilled. Do you see any requirement stating that the code has to follow ES6 syntax? Please show me the piece of code you based your action upon.

If you talked about JavaScript best practices (as required by Microverse), I ***kindly*** suggest you read [this file](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and I think you'll find the fourth point of the General practices enlightening:

> 4. ***IF*** you use ES6, use object destructuring to get the values from an object. This way you can avoid repeating a lot of code.

I usually am very kind to people and I don't feel offended when being given a suggestion or when standing corrected - nor do I ask for a second opinion on a review, but, as sorry as I may be, in this instance I have to do just that and to complain about the fact that you intentionally tried to be unfair to me.

Happy reviewing! :)